app/mailers/concerns/mailable.rb

```diff
@@ -0,0 +1,14 @@
+module Mailable
+  extend ActiveSupport::Concern
+
+  def email_with_name(email, name)
+    # http://stackoverflow.com/a/8106387/358804
+    address = Mail::Address.new(email)
+    address.display_name = name
+    address.format
+  end
+
+  def attach_images
+    attachments.inline['logo.png'] = File.read('app/assets/images/logo.png')
```

Comment:
agreed! I am open to doing that separately; for this PR I am worried that I've already overstepped by boundaries a bit (eg: moving this method) so would prefer to keep changes that are not exactly related separate if I can.